src/SqsConsumer.ts

```diff
@@ -0,0 +1,181 @@
+import * as Sentry from '@sentry/node';
+import { eventConsumer } from './eventConsumer';
+import { EventEmitter } from 'events';
+import { config } from './config';
+import {
+  SQSClient,
+  ReceiveMessageCommand,
+  DeleteMessageCommand,
+  Message,
+  ReceiveMessageCommandOutput,
+  SendMessageCommand,
+} from '@aws-sdk/client-sqs';
+import { setTimeout } from 'timers/promises';
+
+/**
+ * class to poll the SQS message from the snowplow event queue,
+ * and process the message for sending it to snowplow platform.
+ * If we are unable to process the message, we will add it to DLQ
+ * and alert sentry.
+ * We will delete the message after its processed, whether successful or not
+ * from the main queue to prevent queue flooding
+ */
+export class SqsConsumer {
+  readonly sqsClient: SQSClient;
+  static readonly eventName = 'pollSnowplowSqsQueue';
+  constructor(public readonly emitter: EventEmitter, pollOnInit = true) {
+    console.log(`retriving queue`);
```

Comment:
```suggestion
    console.log(`retrieving queue`);
```
normally wouldn't worry about this, but since its in logs and we may want to search or not on it, went ahead